documentation/modules/post/windows/gather/enum_hostfile.rb

```diff
@@ -0,0 +1,45 @@
+## Vulnerable Application
```

Comment:
Just confirming if this is meant to be a `.rb` file? 👀

Reply:
oops!